Userland/Libraries/LibRegex/RegexDebug.h

```diff
@@ -129,8 +129,10 @@ class RegexDebug {
         for (size_t i = 0; i < length; ++i) {
             builder.append('=');
         }
+        auto str = builder.to_string();
+        VERIFY(!str.is_empty());
 
-        fprintf(m_file, "%s\n", builder.to_string().characters());
+        fprintf(m_file, "%s\n", str.characters());
```

Comment:
Makes absolutely no sense to me how this could possibly be null, so I added a VERIFY. .... 

```
In file included from ../Userland/Libraries/LibRegex/Regex.h:30,
                 from ../Userland/Libraries/LibRegex/Tests/Regex.cpp:30:
../Userland/Libraries/LibRegex/RegexDebug.h: In member function 'void regex::RegexDebug::print_header()':
../Userland/Libraries/LibRegex/RegexDebug.h:133:16: error: '%s' directive argument is null [-Werror=format-overflow=]
  133 |         fprintf(m_file, "%s\n", builder.to_string().characters());
      |         ~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
It can't see the definition of `StringBuilder::to_string()`, and `String::characters()` _may_ return `nullptr` (even though `StringBuilder::to_string()` will never return that).

Reply:
Gotcha. I'll keep the VERIFY then, since the compiler can't assume what to_string does and we can't un-confuse it without a check.